### DISCUSSION TITLE
Required Jira Package and Jenkins Plugin Latest HPI File

### DISCUSSION THREAD

**Discussion by vaibhav-rastogi** [Ginger-Automation/Ginger#2236] [Q&A] [answered]
Hello Team, When i am trying to connect JIRA using ALM setting its asking to load configuration package. There is download button which is navigating to Intranet URL could you please help with JIRA configuration package file ?
Also in order to work with Latest ginger version and ginger runner we would need Jenkins Latest HPI file. Could you please help with that ?

**Comment by JinendraGhodke** [comment, marked as answer]
Attaching the Jira package and latest Jenkins plugin

[JiraSettings.zip](https://github.com/Ginger-Automation/Ginger/files/7948756/JiraSettings.zip)
[Ginger_Jenkins_plugin_v3.1.zip](https://github.com/Ginger-Automation/Ginger/files/7948767/Ginger_Jenkins_plugin_v3.1.zip)
